packages/melos/lib/src/command_runner/base.dart

```diff
@@ -123,6 +123,12 @@ abstract class MelosCommand extends Command<void> {
           'matches the other filters. The included packages skip --ignore '
           'and --since checks.',
     );
+
+    argParser.addOption(
+      filterOptionSdkPath,
```

Comment:
I don't think this option should be a filter option. We could add a new `GlobalOptions` object for which the `argParser` is setup in `setupGlobalOptionsParser` and is parsed in `parseGlobalOptions`.

Reply:
Thanks for clarifying. What is a filter option supposed to be used for?

Reply:
The filter options are used to select a subset of the packages in a workspace, for example to only bootstrap Flutter packages and not pure Dart packages you could use `melos bootstrap --flutter`.